checkST2110.js

```diff
@@ -28,12 +28,14 @@ const groupPattern = /a=group:DUP\s+(\S+)\s+(\S+)/;
 const ssrcPattern = /a=ssrc:(\d+)\s/;
 const videoPattern = /video\s+(\d+)(\/\d+)?\s+(RTP\/S?AVP)\s+(\d+)/;
 const rtpmapPattern = /a=rtpmap:(\d+)\s(\S+)\/(\d+)\s*/;
+const bandwidthPattern = /b=([a-zA-Z]+):(\d*\s*$)/;
 const fmtpElement = '([^\\s=;]+)(?:=([^\\s;]+))?';
 const fmtpSeparator = '(?:;\\s*)';
 const fmtpPattern = new RegExp('a=fmtp:(\\d+)\\s*(\\s' + fmtpElement + '(' + fmtpSeparator + fmtpElement + ')*' + fmtpSeparator + '?)?$');
 const fmtpParams = new RegExp(fmtpElement + fmtpSeparator + '?', 'g');
 const integerPattern = /^[1-9]\d*$/;
 const frameRatePattern = /^([1-9]\d*)(?:\/([1-9]\d*))?$/;
+const frameRateAttributePattern = /a=framerate:\d*.\d*$/;
```

Comment:
`*` is zero-or-more, and `.` is any character, so this probably isn't precise enough, it would accept `a=framerate:F00` for example.

Admittedly, RFC 4566 has a horribly vague specification for the `framerate` attribute, but RFC 8866 tightens it up as follows:

```
framerate-value      = non-zero-int-or-real
non-zero-int-or-real = integer / non-zero-real
integer              = POS-DIGIT *DIGIT
non-zero-real        = zero-based-integer "." *DIGIT POS-DIGIT
zero-based-integer   = "0" / integer
POS-DIGIT            = %x31-39 ; 1 - 9
DIGIT                = <DIGIT definition from RFC 5234> ; %x30-39 ; 0 - 9
```

That makes for a reasonably complex regex pattern, and interestingly rejects `a=framerate:50.000` which I suspect we might see in the wild:

```suggestion
const frameRateAttributePattern = /a=framerate:([1-9]\d*|(0|[1-9]\d*)\.\d*[1-9])$/;
```

Reply:
Tested this new pattern and works as per spec. I'll add a helpful hint to the error about the framerate needing to end with 1-9 in the decimal part.